src/cli/lint/eslint.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import { isMainThread } from 'worker_threads';
+
+import chalk from 'chalk';
+
+import { createLogger } from '../../utils/logging';
+import { execWorkerThread, startWorkerThread } from '../../utils/worker';
+import { runESLint } from '../adapter/eslint';
+
+import type { Input } from './types';
+
+export const runESLintInMainThread = ({ debug }: Input) =>
```

Comment:
It's a bit confusing that `runESLintInWorkerThread` calls `runESLintInMainThread`. Maybe `runESlintInCurrentThread` would be more accurate?